clox/functions.py

```diff
@@ -125,6 +126,24 @@ def show_countries_list() -> None:
                                                                     country_code=country_code, country_name=country_code))
 
 
+def show_date_formats_list(date_system: str = "GREGORIAN") -> None:
+    """
+    Show date formats list.
+
+    :param date_system: date system
+    """
+    datetime_lib = datetime
+    example_year = 1990
+    if date_system == "JALALI":
+        datetime_lib = jdatetime
+        example_year = 1368
+    print("Date formats list:\n")
+    example_date = datetime_lib.datetime(year=example_year, month=1, day=1)
```

Comment:
use different value for `month` and `day` (e.g,. a day > 12) so that it's distinguishable in example which one is which.